common/output-model.cpp

```diff
@@ -907,6 +907,27 @@ bool output_model::user_defined_command( std::string command, device_models_list
             }
         }
     }
+    else if( to_lower( command ) == "get-nest" )
+    {
+        user_defined_command_detected = true;
+
+        for( auto && dev : devices )
+        {
+            if( auto dbg = dev.as< rs2::debug_protocol >() )
+            {
+                if( dev.supports( RS2_CAMERA_INFO_FIRMWARE_VERSION ) )
+                {  // Verify minimal version for handling this command
+                    if( ! is_upgradeable( dev.get_info( RS2_CAMERA_INFO_FIRMWARE_VERSION ), "01.05.0.0" ) )
+                    {
+                        std::vector< uint8_t > special_command
+                            = { 'G', 'E', 'T', '-', 'N', 'E', 'S', 'T' };
+                        auto res = dbg.send_and_receive_raw_data( special_command );
```

Comment:
This can throw... what will happen?

Reply:
It will be catched on the viewer side and write a log warning (the whole function is wrapped by try catch from Sergey's code that;s why I didn't change it to error)